rubicon_ml/client/parameter.py

```diff
@@ -22,36 +31,36 @@ class Parameter(Base, TagMixin):
         The experiment that the parameter is logged to.
     """
 
-    def __init__(self, domain, parent):
+    def __init__(self, domain: ParameterDomain, parent: Experiment):
         super().__init__(domain, parent._config)
         self._parent = parent
 
     @property
-    def id(self):
+    def id(self) -> str:
         """Get the parameter's id."""
         return self._domain.id
 
     @property
-    def name(self):
+    def name(self) -> Optional[str]:
         """Get the parameter's name."""
         return self._domain.name
 
     @property
-    def value(self):
+    def value(self) -> Optional[Union[object, float]]:
         """Get the parameter's value."""
-        return self._domain.value
+        return getattr(self._domain, "value", None)
```

Comment:
why is this necessary? a parameter domain should never not have a `value` key

Reply:
Good catch. This was because I didn't hint the `_domain` value properly.